znai-reactjs/src/App.css

```diff
@@ -174,7 +175,7 @@
     --znai-collapse-uncollapse-background-color: var(--znai-snippets-title-background-color);
 
     --znai-inversed-text-color: #bbbbbb;
-    --znai-inversed-background-color: #282f34;
+    --znai-inversed-background-color: #030303;
```

Comment:
did you change it on purpose or was it left over from some experimenting?

Reply:
Think I fixed.